lib/auth/grpcserver.go

```diff
@@ -2163,8 +2163,11 @@ func (g *GRPCServer) DeleteRole(ctx context.Context, req *authpb.DeleteRoleReque
 func doMFAPresenceChallenge(ctx context.Context, actx *grpcContext, stream authpb.AuthService_MaintainSessionPresenceServer, challengeReq *authpb.PresenceMFAChallengeRequest) error {
 	user := actx.User.GetName()
 
+	// TODO(Joerger): Extend SSO MFA support for moderated sessions.
+	var ssoClientRedirectURL string
```

Comment:
This will be implemented in a follow up PR when I can get to it, but it's not a priority over per-session MFA and other MFA methods.